daemon/graphdriver/driver_windows.go

```diff
@@ -0,0 +1,26 @@
+package graphdriver
+
+type DiffDiskDriver interface {
+	Driver
+	CopyDiff(id, sourceId string) error
+}
+
+const (
+	FsMagicWindows = FsMagic(0xa1b1830f)
+)
+
+var (
+	// Slice of drivers that should be used in an order
+	priority = []string{
+		"windows",
+	}
+
+	FsNames = map[FsMagic]string{
+		FsMagicWindows:     "windows",
```

Comment:
hmm would a filesystem name be better here just in case more cow FSes come out in windows area as well?